lambda/file_format.tf

```diff
@@ -0,0 +1,181 @@
+  tags          = var.common_tags
+  environment {
+    variables = {
+      ENVIRONMENT   = local.environment
+      INPUT_QUEUE   = local.file_format_queue_url
+      OUTPUT_QUEUE  = local.api_update_queue_url
+      AUTH_URL      = var.auth_url
+      API_URL       = "${var.api_url}/graphql"
+      CLIENT_ID     = "tdr-backend-checks"
+      CLIENT_SECRET = data.aws_ssm_parameter.backend_checks_client_secret[0].value
+    }
+  }
+  file_system_config {
+    # EFS file system access point ARN
+    arn              = var.file_format_efs_access_point.arn
+    local_mount_path = "/mnt/fileformat"
+  }
+
+  depends_on = [aws_efs_mount_target.target_az_zero, aws_efs_mount_target.target_az_one]
+
+  vpc_config {
+    subnet_ids         = aws_subnet.file_format_private.*.id
+    security_group_ids = [aws_security_group.allow_efs_lambda[count.index].id]
+  }
+
+  lifecycle {
+    ignore_changes = [filename]
+  }
+}
+
+resource "aws_efs_mount_target" "target_az_zero" {
+  count           = local.count_file_format
+  file_system_id  = var.file_system.id
+  subnet_id       = aws_subnet.file_format_private[0].id
+  security_groups = [aws_security_group.mount_target_sg[0].id]
+}
+
+resource "aws_efs_mount_target" "target_az_one" {
+  count           = local.count_file_format
+  file_system_id  = var.file_system.id
+  subnet_id       = aws_subnet.file_format_private[1].id
+  security_groups = [aws_security_group.mount_target_sg[0].id]
+}
+
+resource "aws_security_group" "mount_target_sg" {
+  count       = local.count_file_format
+  name        = "mount-target-security-group"
+  description = "Mount target security group"
+  vpc_id      = data.aws_vpc.current.id
+
+  ingress {
+    description     = "EFS"
+    from_port       = 2049
+    to_port         = 2049
+    protocol        = "tcp"
+    security_groups = [aws_security_group.ecs_run_efs[count.index].id, aws_security_group.allow_efs_lambda[count.index].id]
+  }
+
+  egress {
+    protocol    = "-1"
+    from_port   = 0
+    to_port     = 0
+    cidr_blocks = ["0.0.0.0/0"]
+  }
+
+  tags = merge(
+    var.common_tags,
+    map("Name", "mount-target-outbound-only")
+  )
+}
+
+resource "aws_security_group" "ecs_run_efs" {
+  count       = local.count_file_format
+  name        = "allow-ecs-mount-efs"
+  description = "Allow ECS to mount EFS volume"
+  vpc_id      = data.aws_vpc.current.id
+
+  egress {
+    protocol    = "-1"
+    from_port   = 0
+    to_port     = 0
+    cidr_blocks = ["0.0.0.0/0"]
+  }
+
+  tags = merge(
+    var.common_tags,
+    map("Name", "allow-ecs-mount-efs")
+  )
+}
+
+resource "aws_security_group" "allow_efs_lambda" {
+  count       = local.count_file_format
+  name        = "allow-efs"
+  description = "Allow EFS inbound traffic"
+  vpc_id      = data.aws_vpc.current.id
+
+  egress {
+    protocol    = "-1"
+    from_port   = 0
+    to_port     = 0
+    cidr_blocks = ["0.0.0.0/0"]
+  }
+
+  tags = merge(
+    var.common_tags,
+    map("Name", "file-format-lambda-allow-efs")
```

Comment:
Should the product be prefixed to the "Name" property?

Reply:
Yeah, I've added that in.